packages/forms/src/UIForm/fields/Enumeration/EnumerationWidget.js

```diff
@@ -1050,6 +1057,7 @@ if (process.env.NODE_ENV !== 'production') {
 		onTrigger: PropTypes.func.isRequired,
 		properties: PropTypes.object,
 		schema: PropTypes.object,
+		enableSkip: PropTypes.bool,
```

Comment:
The name is not clear enough, I suggest a rename, don't know if you can find better?
```suggestion
		skipCommas: PropTypes.bool,
```